spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe FoodFacade, :vcr do 
+
+    it 'returns food data of the searched food' do 
+        search = FoodFacade.food_finder("sweet potatoes") 
+
+        expect(search).to be_a(Array)
```

Comment:
I'd expect an assertion here about what type of objects are in the array (i.e. Food objects)